src/main/resources/Help/Code/Translations.fr.xml

```diff
@@ -67,7 +67,7 @@ help.applications.title1=Exemples
 help.browse.label=Parcourir
 help.description1=Wiki offre de nombreuses fonctionnalités puissantes qui vous permettront de créer, organiser et trouver du contenu. Cette section vous montrera comment utiliser les fonctionnalités de base d'XWiki.
 help.description2=Félicitations pour l'acquisition de votre instance. XWiki est très flexible et peut être utilisé pour créer des bases de connaissances, des intranets ou tout autre type d'application métier. Ceci est possible grâce à ses grandes capacités de personnalisation, qui commencent par les pages, de manière simple ou plus avancée.
-help.edit.description=Utilisez un éditeur de texte {0}WYSIWYG{1} ou la {2}syntaxe{3} pour créer des pages wiki enrichies.
+help.edit.description=Utilisez un éditeur de texte {0}WYSIWYG{1} ou la {2}syntaxe{1} pour créer des pages wiki enrichies.
```

Comment:
I'm not a huge fan, I guess it would be cleaner to use 4 parameters, even if 1 and 3 would be the same. I haven't noticed it in the first PR. Anyway, now it's done like that.

Now I'm wondering if it wouldn't be better to fix it directly in weblate, I guess it will take into account automatically the changes made to the translations in the repo but I wonder if we could have some sync issue, WDYT @tmortagne ?

Reply:
> I guess it would be cleaner to use 4 parameters

It makes sense to use 4 parameters only if there is a case where you want to use different values. If there's no such case then using less parameters and reusing their values is better (and less error-prone).

Reply:
> (and less error-prone)

apparently it's not :) 
looks more natural to write `{0}...{1} ... {2}...{3}` than `{0}...{1} ... {2}...{1}` especially for people who translate I think.

Reply:
The parameters are indexed for a reason: to be able to use them in **any order** and **multiple times**. Otherwise it would have been easier to write:

    Utilisez un éditeur de texte {}WYSIWYG{} ou la {}syntaxe{} pour ...

If the parameters are always in order and they appear only once then the index is redundant.

For me this is close to writing a function with 4 parameters that expects the forth parameter to have the same value as the second. If the function has this expectation then why not use 3 parameters instead.